pkg/cmd/pulumi/up.go

```diff
@@ -40,7 +40,8 @@ import (
 
 // The default number of parallel resource operations to run at once during an update, if --parallel is unset.
 // See https://github.com/pulumi/pulumi/issues/14989 for context around the cpu * 4 choice.
-var defaultParallel = runtime.NumCPU() * 4
+var defaultParallel = int32(runtime.NumCPU()) * 4 //nolint:gosec // NumCPU is an int32 internally,
+//                                                                  but the NumCPU function returns an int.
```

Comment:
```suggestion
//nolint:gosec // NumCPU is an int32 internally, but the NumCPU function returns an int.
var defaultParallel = int32(runtime.NumCPU()) * 4
```

Very minor nitpick.  If you prefer the other way that's also fine with me :)